### PR TITLE
Add oneaccess SSH

### PR DESCRIPTION
**Pull request by mwallraf**
Separate SSH driver for OneAccess OneOS devices.  These are very similar to Cisco but require a separate EOL.

### CODE REVIEW

File: netmiko/oneaccess/oneaccess_oneos.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+from netmiko.base_connection import BaseConnection
+
+
+class OneaccessOneosSSH(CiscoSSHConnection):
+    def __init__(self, *args, **kwargs):
+        default_enter = kwargs.get("default_enter")
+        kwargs["default_enter"] = "\r\n" if default_enter is None else default_enter
+        super(OneaccessOneosSSH, self).__init__(*args, **kwargs)
+
+class OneaccessOneosTelnet(BaseConnection):
```
Comment (ktbyers) [inline netmiko/oneaccess/oneaccess_oneos.py]: Why is this inheriting from `BaseConnection`?

I would probably have expected there to be a `OneaccessOneosBase(CiscoSSHConnection)` and then both OneaccessOneosTelnet and OneaccessOneosSSH both inherit from that?

**Comment by ktbyers** [comment]
It looks like you didn't actually implement anything for telnet. Which is fine, but then you should just remove it and only have SSH.

Also any possibility you can run the 'show' and 'config' unit tests using your code against a test one access device?

**Comment by mwallraf** [comment]
The telnet connection is still work in progress but probably I should remove it for now to keep the code more clear. And in that case the BaseConnection import is not required either.

I will update and add the unit tests as well.

**Comment by ktbyers** [comment]
Okay, sounds good.

**Comment by mwallraf** [comment]
Hi,

I've added the unit tests in the sample yml files and ran them against a test device.
Also the telnet function has been added.

device types:

oneaccess_oneos
oneaccess_oneos_telnet


Starting tests...good luck:
===================================================================================================================== test session starts ======================================================================================================================
platform linux -- Python 3.5.3, pytest-2.9.1, py-1.7.0, pluggy-0.3.1 -- /home/mwallraf/netmiko/venv/bin/python3.5
cachedir: ../.cache
rootdir: /home/mwallraf/netmiko/oneaccess/netmiko, inifile:
plugins: pylama-7.4.3
collected 12 items

test_netmiko_show.py::test_disable_paging PASSED
test_netmiko_show.py::test_ssh_connect PASSED
test_netmiko_show.py::test_ssh_connect_cm PASSED
test_netmiko_show.py::test_send_command_timing PASSED
test_netmiko_show.py::test_send_command_expect PASSED
test_netmiko_show.py::test_base_prompt PASSED
test_netmiko_show.py::test_strip_prompt PASSED
test_netmiko_show.py::test_strip_command PASSED
test_netmiko_show.py::test_normalize_linefeeds PASSED
test_netmiko_show.py::test_clear_buffer PASSED
test_netmiko_show.py::test_enable_mode PASSED
test_netmiko_show.py::test_disconnect PASSED

================================================================================================================== 12 passed in 47.56 seconds ==================================================================================================================
===================================================================================================================== test session starts ======================================================================================================================
platform linux -- Python 3.5.3, pytest-2.9.1, py-1.7.0, pluggy-0.3.1 -- /home/mwallraf/netmiko/venv/bin/python3.5
cachedir: ../.cache
rootdir: /home/mwallraf/netmiko/oneaccess/netmiko, inifile:
plugins: pylama-7.4.3
collected 7 items

test_netmiko_config.py::test_ssh_connect PASSED
test_netmiko_config.py::test_enable_mode PASSED
test_netmiko_config.py::test_config_mode PASSED
test_netmiko_config.py::test_exit_config_mode PASSED
test_netmiko_config.py::test_command_set PASSED
test_netmiko_config.py::test_commands_from_file PASSED
test_netmiko_config.py::test_disconnect PASSED

================================================================================================================== 7 passed in 53.34 seconds ===================================================================================================================
(venv) mwallraf@camelus:~/netmiko/oneaccess/netmiko/tests$
(venv) mwallraf@camelus:~/netmiko/oneaccess/netmiko/tests$
(venv) mwallraf@camelus:~/netmiko/oneaccess/netmiko/tests$
(venv) mwallraf@camelus:~/netmiko/oneaccess/netmiko/tests$
(venv) mwallraf@camelus:~/netmiko/oneaccess/netmiko/tests$ test_oneaccess_oneos.sh
-bash: test_oneaccess_oneos.sh: command not found
(venv) mwallraf@camelus:~/netmiko/oneaccess/netmiko/tests$ sh test_oneaccess_oneos.sh
Starting tests...good luck:
===================================================================================================================== test session starts ======================================================================================================================
platform linux -- Python 3.5.3, pytest-2.9.1, py-1.7.0, pluggy-0.3.1 -- /home/mwallraf/netmiko/venv/bin/python3.5
cachedir: ../.cache
rootdir: /home/mwallraf/netmiko/oneaccess/netmiko, inifile:
plugins: pylama-7.4.3
collected 12 items

test_netmiko_show.py::test_disable_paging PASSED
test_netmiko_show.py::test_ssh_connect PASSED
test_netmiko_show.py::test_ssh_connect_cm PASSED
test_netmiko_show.py::test_send_command_timing PASSED
test_netmiko_show.py::test_send_command_expect PASSED
test_netmiko_show.py::test_base_prompt PASSED
test_netmiko_show.py::test_strip_prompt PASSED
test_netmiko_show.py::test_strip_command PASSED
test_netmiko_show.py::test_normalize_linefeeds PASSED
test_netmiko_show.py::test_clear_buffer PASSED
test_netmiko_show.py::test_enable_mode PASSED
test_netmiko_show.py::test_disconnect PASSED

================================================================================================================== 12 passed in 34.85 seconds ==================================================================================================================
===================================================================================================================== test session starts ======================================================================================================================
platform linux -- Python 3.5.3, pytest-2.9.1, py-1.7.0, pluggy-0.3.1 -- /home/mwallraf/netmiko/venv/bin/python3.5
cachedir: ../.cache
rootdir: /home/mwallraf/netmiko/oneaccess/netmiko, inifile:
plugins: pylama-7.4.3
collected 7 items

test_netmiko_config.py::test_ssh_connect PASSED
test_netmiko_config.py::test_enable_mode PASSED
test_netmiko_config.py::test_config_mode PASSED
test_netmiko_config.py::test_exit_config_mode PASSED
test_netmiko_config.py::test_command_set PASSED
test_netmiko_config.py::test_commands_from_file PASSED
test_netmiko_config.py::test_disconnect PASSED

================================================================================================================== 7 passed in 47.86 seconds ===================================================================================================================

**Comment by ktbyers** [comment]
FYI, I made some minor changes to some of the naming.

**Comment by ktbyers** [comment]
@mwallraf Good job on this.

**Pull request merged**